core/src/main/java/com/scalar/db/transaction/consensuscommit/ParallelExecutor.java

```diff
@@ -23,7 +23,7 @@ public class ParallelExecutor {
 
   @FunctionalInterface
   public interface ParallelExecutorTask {
-    void run() throws ExecutionException, CommitConflictException;
+    void run() throws ExecutionException, ValidationConflictException;
```

Comment:
Made this throw `ValidationConflictException` instead of `CommitConflictException`.